examples/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This file shouldn't be needed. All dependencies can go into dev-dependencies in the root package.json